olive/passes/onnx/transformer_optimization.py

```diff
@@ -157,13 +157,12 @@ def _run_for_config(
 
         if model.model_attributes:
             model_config = model.model_attributes
-            input_model_type = model_config.get("model_type", "")
-            _model_type = MODEL_TYPE_MAPPING.get(input_model_type, input_model_type)
-            run_config["model_type"] = run_config["model_type"] or _model_type
-            assert run_config["model_type"] in transformers_optimizer.MODEL_TYPES, (
-                f"Unsupported model type: {run_config['model_type']}, please select one from "
-                "{transformers_optimizer.MODEL_TYPES} which need to be set under OrtTransformersOptimization.config"
-            )
+            input_model_type = model_config.get("model_type")
```

Comment:
not an issue but I think the original lines 160-162 do the same thing as the new `160-165`? `""` behaves the same as `None` in this use case. 

The only thing incorrect was that the final `model_type` check should have been outside this if condition

Reply:
Yes, your are correct.

For the model_type variable, it is fine since it is only used when model.model_attributes have values.